src/h5web/toolbar/OverflowMenu.tsx

```diff
@@ -46,18 +46,20 @@ function OverflowMenu(props: Props): ReactElement {
         </span>
       </button>
 
-      <ul
+      <div
```

Comment:
This new wrapper is not technically needed for the overflow menu, but is required for the tooltip in order to have an invisible padding gap between the toolbar and the popup that doesn't "lose" the hover (unlike a gap that would created by absolutely positioning the popup away from the toolbar).